New Theme (Flowing)

Raamzeez opened the pull request:
I made a new theme called flowing which focuses on making the UI more cleaner with having rounded buttons that also give off a shadow when you hover them. Additionally, there are a lot of gradients sprinkled throughout the whole site. I wanted to have the effect of colors flowing by adding many gradients. Please let me know if there are any changes that can be made so that it can hopefully be added to the public site. Thanks.

<img width="498" alt="Screen Shot 2021-07-19 at 3 41 40 PM" src="https://user-images.githubusercontent.com/37488506/126217602-0d42d992-42d6-4994-a5e5-dd019f881e1e.png">

<img width="1575" alt="Screen Shot 2021-07-19 at 3 41 21 PM" src="https://user-images.githubusercontent.com/37488506/126217560-3bee3fc6-87a4-49e8-af20-dc64f1545688.png">

<img width="1920" alt="Screen Shot 2021-07-19 at 3 40 55 PM" src="https://user-images.githubusercontent.com/37488506/126217503-dac4b68d-ffa6-488a-b8ee-703b57f8b6c7.png">

<img width="1920" alt="Screen Shot 2021-07-19 at 3 40 35 PM" src="https://user-images.githubusercontent.com/37488506/126217463-6a408f77-c32b-45be-9dbc-2938d022ddbb.png">

johnnyapol commented on site/src/quacs-rs/Cargo.toml:
```diff
@@ -25,7 +25,7 @@ console_error_panic_hook = { version = "0.1.6", optional = true }
 
 [package.metadata.wasm-pack.profile.release]
-wasm-opt = ["-O3"]
+wasm-opt = false
```
I understand you need this for your local builds, but this should not be committed to the repo

Raamzeez replied:
Yes you are right sorry about that. I did change this but it is on my forked repository. Do I have to make another pull request or is there an easy way for this to be edited? Sorry I am not thoroughly familiar with GitHub.

johnnyapol replied:
Looks like you fixed it on your next commit. Looks good now.

Raamzeez replied:
Ignore my previous comment if the changes have already been made to the file

Raamzeez replied:
Ok sounds good, thanks!

johnnyapol commented:
Other than the Cargo.toml, theme looks nice!

benjaminrsherman reviewed (changes requested):
When you're addressing the specific comments, please make sure to keep the different themes' css files with the same layout.  The only difference between the files should be the values of the variables and which lines are commented out, otherwise they should be the same (i.e. whitespace should match, the order of variables should match, etc.)

Raamzeez commented on site/src/assets/styles/colors.css:
```diff
@@ -47,34 +47,26 @@
     --taken-course: #228C3A;
     --not-taken-course: #dc3545;
     --overlay-background: #f8f9fa;
-
     --num-calendar-colors: 7;
-
     --calendar-bg-color-0: #ffe9cf;
     --calendar-border-color-0: #ff9332;
     --calendar-text-color-0: #000000;
-
     --calendar-bg-color-1: #ffd4df;
     --calendar-border-color-1: #ff2066;
     --calendar-text-color-1: #000000;
-
     --calendar-bg-color-2: #ceeffc;
     --calendar-border-color-2: #00aff2;
     --calendar-text-color-2: #000000;
-
     --calendar-bg-color-3: #fff4d0;
     --calendar-border-color-3: #ffcb45;
     --calendar-text-color-3: #000000;
-
     --calendar-bg-color-4: #dcf7da;
     --calendar-border-color-4: #48da58;
     --calendar-text-color-4: #000000;
-
     --calendar-bg-color-5: #f7e2f7;
     --calendar-border-color-5: #d373da;
     --calendar-text-color-5: #000000;
-
```
Sorry this was because I was using a formatter and it automatically got rid of these spaces. I can revert them easily.

Raamzeez commented:
@benjaminrsherman I made a new commit a few minutes ago. Please check it out whenever you get a chance. Thanks!

Raamzeez commented:
@benjaminsherman I went back again and removed the whitespaces that I found and added the variables to all the themes. Sorry for the late commit because I have been busy and also sorry if I missed anything in my changes. If there is anything more that I have to change then I will happily do that. I had to go back to my previous commits and see the things that I had changed from about a week ago so I might have missed a few things.
